### DISCUSSION TITLE
How to listen for messages?

### DISCUSSION THREAD

**Discussion by christau** [Cosium/matrix-communication-client#53] [Q&A]
Hi all,
I want to create a chat bot. How do I listen for new messages?
So far I know that 
```
events = room.fetchEventPage(null, null, null, null);
```
could be used. But is there any way that I can attach a listener somewhere to get informed about new messages, or is this just a 'busy polling' kinda thing to do?

Many thanks in advance for any input.

**Comment by reda-alaoui** [comment]
Hello,
Not for now. PRs welcome.
